modelkit/core/library.py

```diff
@@ -63,6 +63,8 @@ def __init__(
         self.assetsmanager_settings = assetsmanager_settings or {}
         self._override_assets_manager = None
         self._lazy_loading = self.settings.lazy_loading
+        if models is None:
+            models = os.environ.get("MODELKIT_DEFAULT_PACKAGE", None)
```

Comment:
NB: the explicit default to None is not useful

Reply:
Done thanks